### PR TITLE
Improve and move paragraph layout and logic to own component

### PR DESCRIPTION
**Pull request by takvamborgen**

### CODE REVIEW

File: src/pages/prep_and_analysis/note/NoteParagraph.js [hunk 1]
```diff
@@ -1,16 +1,121 @@
...
+        }
+      })
+      .catch( () => {
+        // Zeppelin returns 500 if a paragraph fails due to error in the code. This should not
+        // be presented to the user as an application error. The paragraph result will display the error
+        getParagraph()
+      })
+  }
+
+  const getParagraph = () => {
```
Comment (bjornandre) [inline src/pages/prep_and_analysis/note/NoteParagraph.js]: A getter should not have side effects. Perhaps the method should be called `loadParagraph` instead?

File: src/pages/prep_and_analysis/note/NoteParagraph.js [hunk 2]
```diff
@@ -1,16 +1,121 @@
...
+        if (result.status === 'OK') {
+          setParagraph(result.body)
+        } else {
+          context.setNotification(true, NOTIFICATION_TYPE.ERROR,
+            context.getLocalizedText(UI.GENERIC_ERROR) + ' - ' + result.status)
+        }
+      })
+      .catch( error => {
+        context.setNotification(true, NOTIFICATION_TYPE.ERROR, error.text)
+        getParagraph()
```
Comment (bjornandre) [inline src/pages/prep_and_analysis/note/NoteParagraph.js]: This is confusing. Why does the method call itself?

File: src/App.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ class App extends Component {
 
 // Callback from UserManager
   onUserLoaded(user) {
-    console.log("User loaded", user)
+    console.log("User loaded", user.profile)
```
Comment (bjornandre) [inline src/App.js]: 👍

File: src/pages/prep_and_analysis/note/NoteParagraph.js (hunk 1 above)
Comment (takvamborgen) [reply src/pages/prep_and_analysis/note/NoteParagraph.js]: I agree. Automatic naming (in my head) due to it calling GET. I will change it.

File: src/pages/prep_and_analysis/note/NoteParagraph.js (hunk 2 above)
Comment (takvamborgen) [reply src/pages/prep_and_analysis/note/NoteParagraph.js]: Try again? :) No, copy-paste error. Will fix.

**Pull request merged**